manager/controllers/app/m4dapplication_controller.go

```diff
@@ -333,29 +331,6 @@ func AnalyzeError(app *app.M4DApplication, log logr.Logger, assetID string, err
 	return err
 }
 
-// CreateNamespace creates a namespace in which the blueprint and the relevant resources will be running
-// It stores the generated namespace name inside app status
-func (r *M4DApplicationReconciler) CreateNamespace(app *app.M4DApplication) error {
-	genNamespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{GenerateName: "m4d-"}}
-	genNamespace.Labels = map[string]string{
-		"m4d.ibm.com.owner": app.Namespace + "." + app.Name,
-	}
-	if err := r.Create(context.Background(), genNamespace); err != nil {
-		return err
-	}
-	r.Log.V(0).Info("Created namespace " + genNamespace.Name + " for " + app.Namespace + "/" + app.Name)
-	app.Status.BlueprintNamespace = genNamespace.Name
-	return nil
-}
-
-// DeleteNamespace deletes the blueprint namespace upon blueprint deletion
-func (r *M4DApplicationReconciler) DeleteNamespace(name string) error {
-	return r.Delete(context.Background(), &corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
-		}})
-}
-
 // Helper functions to manage conditions
 func resetConditions(application *app.M4DApplication) {
```

Comment:
Can be done in different PR, but I think the error and conditions code should be separated out (into utils?) and detached from m4dapplication

Reply:
Done.